compiler/wasm/build-fixtures.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cd test/fixtures && cd simple && nargo compile && cd ../with-deps && nargo compile && cd ../noir-contract && nargo compile && cd ../../
```

Comment:
We can remove the `cd`-ing with `nargo compile --program-dir ./test/fixtures/simple`, etc.

Reply:
Done!